src/main/java/lesson03/conditions/DaysOfMonth.java

```diff
@@ -0,0 +1,63 @@
+package lesson03.conditions;
+/*
+Task 4 (DaysOfMonth)
+Определить, сколько дней в месяце в не високосный год. Не использовать классы из стандартной библиотеки.
+Использовать конструкцию switch-case.
+Ограничения: month = 1 … 12.
+int daysOfMonth(int month)
+Входные аргументы int month
+Тип результата int
+ */
+
+import utils.errorhandling.ExceptionHandling;
+
+public class DaysOfMonth {
+    public static void main (String[] args) {
+        System.out.println(daysOfMonth(13));
+    }
+
+    public static int daysOfMonth(int month){
+        int amountOfDays = 0;
+        switch (month) {
+            case 1:
+                amountOfDays = 31;
+                break;
+            case 2:
+                amountOfDays = 28;
+                break;
+            case 3:
```

Comment:
Try to group similar switch statements:
case 1:
case 3:
amountOfDays = 31;
break;

Reply:
Good idea! Thanks for pointing me on this!!! Fixed that!